standalone_tests/artifact_storage.py

```diff
@@ -0,0 +1,74 @@
+import random
+import os
+import sys
+import tempfile
+
+import wandb
+
+def artifact_with_various_paths():
+    art = wandb.Artifact(type='artsy', name='my artys')
+
+    # internal file
+    with open('random.txt', 'w') as f:
```

Comment:
nit: might want to put this all in a artifact_storage directory just to keep this a bit cleaner.